content/documents/message-senders.mdx

```diff
@@ -590,6 +702,38 @@ If you are building a custom message sender it is necessary to understand what k
   </tbody>
 </table>
 
+### Event Body for OrderReturn Events
+<table>
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Order</td>
+      <td>The <Link to="/api-reference/orders-and-fulfillment/orders/get#response-body">order</Link> that was submitted</td>
+    </tr>
+    <tr>
+      <td>Approvals</td>
+      <td>The array of <Link to="/api-reference/orders-and-fulfillment/orders/list-approvals#response-body">order approvals</Link> for the order</td>
+    </tr>
+    <tr>
+      <td>LineItems</td>
+      <td>The array of <Link to="api-reference/orders-and-fulfillment/line-items/get#response-body">line items</Link> for the orders</td>
+    </tr>
+    <tr>
+      <td>Products</td>
+      <td>The array of <Link to="/api-reference/product-catalogs/products/get#response-body">products</Link> for the order</td>
+    </tr>
+    <tr>
+      <td>OrderReturn</td>
+      <td>The<Link to="/api-reference/orders-and-fulfillment/order-returns/get#response-body">order return</Link> that was submitted</td>
```

Comment:
quick typo fix- could you add a space after 'The'?

Reply:
updated